docs/changelog-fragments/640.doc.rst

```diff
@@ -0,0 +1,2 @@
+Added a Communication section to the README.rst
```

Comment:
This might need to be improved more. It'll be more evident once RTD is fixed.
```suggestion
Added a Communication section to :file:`README.rst`
```